### PR TITLE
Fix: Move dynamic editor CSS to upload directory.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- Trello - https://trello.com/c/E6bfhMaK/358-uag-warning-for-custom-block-file

### Types of changes

- Bug fix (non-breaking change which fixes an issue

### How has this been tested?

- Click on Activate/deactivate the block and check custom-style-blocks.css is created or not in wp-uploads/uag-plugin/custom-style-blocks.css
- Go To the edit page and check by inspecting the page and search  `custom-style-blocks.css` to check it is loading or not.
- Screencast of testing - https://share.getcloudapp.com/E0uAE288

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: classes/class-uagb-admin-helper.php [hunk 1]
```diff
@@ -275,7 +275,8 @@ public static function create_specific_stylesheet() {
 				}
 			}
 
-			$combined_path = UAGB_DIR . 'assets/css/custom-style-blocks.css';
+			$wp_upload_dir = wp_upload_dir( null, false );
```
Comment (RajkiranBagal) [inline classes/class-uagb-admin-helper.php:278]: We can use our `get_uag_upload_dir_path` function to get the Upload DIR Path.

File: classes/class-uagb-scripts-utils.php [hunk 2]
```diff
@@ -106,10 +106,12 @@ public static function enqueue_blocks_dependency_both() {
 	 */
 	public static function enqueue_blocks_styles() {
 
-		if ( file_exists( UAGB_DIR . 'assets/css/custom-style-blocks.css' ) ) {
+		$wp_upload_dir = wp_upload_dir( null, false );
```
Comment (RajkiranBagal) [inline classes/class-uagb-scripts-utils.php:109]: We can use our `get_uag_upload_dir_path` function to get the Upload DIR Path.

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

**Comment by RajkiranBagal** [comment]
Need to update here too -  https://share.getcloudapp.com/2NuqrddB

**Comment by RajkiranBagal** [comment]
- Create a task for this in Trello, Add a proper description of why we are doing this in Task as well as in PR, Add Proper test cases for Tester to test, Add proper PR prefix & Title. 
 
 These things are very important for future references of PR's & should be done by default.

File: classes/class-uagb-scripts-utils.php [hunk 3]
```diff
@@ -106,10 +106,12 @@ public static function enqueue_blocks_dependency_both() {
...
 	public static function enqueue_blocks_styles() {
 
-		if ( file_exists( UAGB_DIR . 'assets/css/custom-style-blocks.css' ) ) {
+		$wp_upload_dir = UAGB_Helper::get_uag_upload_dir_path();
+
+		if ( file_exists( $wp_upload_dir . '/custom-style-blocks.css' ) ) {
 			wp_enqueue_style(
 				'uagb-block-css', // Handle.
-				UAGB_URL . 'assets/css/custom-style-blocks.css', // Block style CSS.
+				$wp_upload_dir . '/custom-style-blocks.css', // Block style CSS.
```
Comment (sandeshjangam) [inline classes/class-uagb-scripts-utils.php:114]: Need url here.

File: classes/class-uagb-admin-helper.php (hunk 1 above)
Comment (AbhijitNage123) [reply classes/class-uagb-admin-helper.php]: Done

File: classes/class-uagb-scripts-utils.php (hunk 2 above)
Comment (AbhijitNage123) [reply classes/class-uagb-scripts-utils.php]: Done. Use URL instead of dir for enqueue

File: classes/class-uagb-scripts-utils.php (hunk 3 above)
Comment (AbhijitNage123) [reply classes/class-uagb-scripts-utils.php]: Updated.

**Review by RajkiranBagal — APPROVED** [review]

**Review by sandeshjangam — APPROVED** [review]

**Pull request merged**